src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -210,6 +210,7 @@ export default {
 			return this.selfIsOwnerOrModerator
 				&& this.conversation.type !== CONVERSATION.TYPE.ONE_TO_ONE
 				&& this.conversation.type !== CONVERSATION.TYPE.ONE_TO_ONE_FORMER
+				&& this.conversation.type !== CONVERSATION.TYPE.NOTE_TO_SELF
```

Comment:
Let's only allow it in the supported types explicitly
```suggestion
				&& (this.conversation.type === CONVERSATION.TYPE.GROUP
					|| this.conversation.type === CONVERSATION.TYPE.PUBLIC)
```